wakepy/methods/wsl.py

```diff
@@ -0,0 +1,100 @@
+import enum
+from abc import ABC, abstractmethod
+from subprocess import Popen, PIPE
+from typing import List
+
+from wakepy.core import DbusAdapter, Method, ModeName, PlatformName
+
+class Flags(enum.StrEnum):
+    KEEP_RUNNING = "$ES_CONTINUOUS -bor $ES_SYSTEM_REQUIRED"
+    KEEP_PRESENTING = "$ES_CONTINUOUS -bor $ES_SYSTEM_REQUIRED -bor $ES_DISPLAY_REQUIRED"
+    RELEASE = "$ES_CONTINUOUS"
+
+
+class WslSetThreadExecutionState(Method, ABC):
+    """This is a method which calls the SetThreadExecutionState function from
```

Comment:
Thanks for keeping a good level of docstrings / comments!

I've been using a strict line limits of 79 for comments and dosctrings, and 88 (from black) for formatting the code. I guess I should write those down to some contribution guide, and probably add some tests for the comments line length. 

If you're using vs code, you could add this to your workspace settings:

```
  "editor.rulers": [
    {
      "column": 79,
      "color": "#664766"
    },
    {
      "column": 88,
      "color": "#7e4343"
    },
  ],
```

I have a laptop with very small screen and these line lengths make it possible to see two tabs of code side-by-side :)